### ISSUE TITLE
Current `dev` branch compilation falis on Ubuntu artful (17.10)

### ISSUE THREAD

**Issue by sickpig** [gandrewstone/bitcoin#950]
The problem seems to be related to the new logging system. What is strange is that it works both on Ubuntu 14.04 and 16.04. This is the error message:

```
  CXX      libbitcoin_server_a-init.o                                                                                                                                                     [8/9497]
httpserver.cpp: In function ‘bool InitHTTPServer()’:                                             
httpserver.cpp:413:27: error: ‘LIBEVENT’ was not declared in this scope                          
     if (LogAcceptCategory(LIBEVENT))                                                            
                           ^~~~~~~~             
httpserver.cpp:413:27: note: suggested alternative:                                              
In file included from sync.h:11:0,              
                 from httpserver.cpp:12:                                                         
util.h:128:5: note:   ‘LIBEVENT’                                                                 
     LIBEVENT = 0x40,                           
     ^~~~~~~~                                   
httpserver.cpp:413:9: error: ‘LogAcceptCategory’ was not declared in this scope                  
     if (LogAcceptCategory(LIBEVENT))           
         ^~~~~~~~~~~~~~~~~                      
httpserver.cpp:413:9: note: suggested alternative:                                               
In file included from sync.h:11:0,              
                 from httpserver.cpp:12:        
util.h:175:13: note:   ‘Logging::LogAcceptCategory’                                              
 inline bool LogAcceptCategory(uint64_t category) { return (categoriesEnabled & category); }     
             ^~~~~~~~~~~~~~~~~                  
Makefile:5553: recipe for target 'libbitcoin_server_a-httpserver.o' failed
 ```

wonder if @Ionoami has some idea of what is going on.

**Comment by sickpig** [comment]
fixed by #957

**Comment by sickpig** [comment]
closing now that #957 is merged
